### DISCUSSION TITLE
Matching between Firely versions and FHIR versions

### DISCUSSION THREAD

**Discussion by luisgmgouveia** [FirelyTeam/firely-net-sdk#2128] [Q&A] [answered]
Hi,

I'm using Firely for 1 year now. It's great, I'm very happy with it!

I'm contacting you because you don't seem to publish a table where we can see the correspondence between Firely versions and FHIR versions. Something like this:
https://hapifhir.io/hapi-fhir/docs/getting_started/versions.html

For instance:
1 - I believe I'm using FHIR 4.6.0 with Firely 3.7.0, but how can I confirm it?
2 - Which is the latest Firely version that is still compatible with FHIR 4.6.0? I would like to install the latest stable version (Firely 4.0.0) but I'm afraid it's already following FHIR 5.0.0 specs.

Thank you!

**Comment by mbaltus** [comment]
The name of the package indicates which FHIR version is supported by that library. So, for example, if you have included [Hl7.Fhir.R4B](https://www.nuget.org/packages/Hl7.Fhir.R4B) you are working with FHIR R4B (also referred to as FHIR version 4.6.0). So you can update that library without having to be worried that you get support for a different FHIR version. Please do make sure to read the change log, to see if updating the library may have consequences for your code.

The table of the different packages - each supporting 1 specific FHIR version - can be found here: https://github.com/FirelyTeam/firely-net-sdk#getting-started

**Reply by luisgmgouveia** [reply]
Hi,

Thank you for the quick answer. I'm sorry but I would like to ask 3 other questions:

1 - Can you provide me a link where I can confirm FHIR R4B is equivalent to 4.6.0 ?
2 - I am using Firely 3.7.0 R5. Which FHIR version is this if not 4.6.0?
3 - I know your R5 branch is compatible with .NET Core. Is R4B also compatible with .NET Core?

Thank you!

**Reply by mbaltus** [reply]
1. Yes, but I do apologize because R4B is actually 4.3.0 and NOT 4.6.0 - I did not realize it when I answered before. This can be found here: http://hl7.org/fhir/directory.html. 4.6.0 is the FHIR R5 preview.
2. See 1. for my mistake between 4.3.0 and 4.6.0. But if you use Hl7.Fhir.R5, it is probably the 5.0.0-snapshot1 version. @marcovisserFurore Is there anywhere we could check this?
3. Yes, all the Hl7.Fhir libraries are .NET Core compatible. You can also check on https://www.nuget.org/packages/.

**Reply by luisgmgouveia** [reply]
Thanks!

**Comment by marcovisserFurore** [comment, marked as answer]
You can see the relationship between the Firely .NET SDK version and the FHIR version [here](https://docs.fire.ly/) (last line).

Next week we are going to release a new version of the Firely .NET SDK (version 4.1) which shall contain the new official FHIR R4B (4.3.0), which has been released by Hl7 at the end of last month.

**Reply by luisgmgouveia** [reply]
Thanks!
